paimon-core/src/main/java/org/apache/paimon/io/DataFileMeta.java

```diff
@@ -135,7 +135,8 @@ public static DataFileMeta forAppend(
             List<String> extraFiles,
```

Comment:
rename method should be changed to new path.

Reply:
done